src/scalacheck/org/scalacheck/Prop.scala

```diff
@@ -496,7 +496,7 @@ object Prop {
 
   /**
    * This handles situations where we have a starting seed in our
-   * parameters.
+   * paramters.
```

Comment:
!

Reply:
Yeah, there were commits like bc0b8dcd07 that made fixes with a tool that should have been submitted upstream.